src/test/java/com/wikia/webdriver/testcases/discussions/UploadingImageTests.java

```diff
@@ -34,11 +38,15 @@
     "Deleted image should not be visible in new post";
   private static final String POST_UNSUPPORTED_IMAGE_NOT_VISIBLE =
     "Unsupported image should not be visible in new post";
+  private static final String POST_OVERWRITTEN_OPENGRAPH_NOT_VISIBLE =
```

Comment:
Have you thought about putting it to inner static class? It may be easier to reuse them in other tests.

Reply:
no need, the messages are specific for test group, which is in this case a single class